admin_app/src/app/dashboard/components/TabPanel.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Box from "@mui/material/Box";
+
+interface TabPanelProps {
+  tabValue: number;
+  handleChange: (event: React.SyntheticEvent, newValue: number) => void;
+}
+
+const TabPanel: React.FC<TabPanelProps> = ({ tabValue, handleChange }) => {
+  const tabLabels = ["Last 24 hours", "Last week", "Last month", "Last year"];
```

Comment:
Shouldn't this be in `index.ts rather than here

Reply:
I'll end up moving this when I build the page. Thanks!